recipe/meta.yaml

```diff
@@ -1,16 +1,25 @@
-{% set version = "1.18.1" %}
+{% set name = "c-ares" %}
+{% set version = "1.19.0" %}
 
 package:
-  name: c-ares
+  name: {{ name|lower }}
```

Comment:
should this be called c-ares-split or similar?

Reply:
You are right. I renamed it as `c-ares-split`